LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 NovelRT
```

Comment:
nit: `Copyright (c) NovelRT and Contributors`

By default, contributors keep their copyright unless they've signed it away via a a CLA (code license agreement).

It would likely also be beneficial to setup a CLA so that way you can make changes later and so people can't revoke contributions based on copyright grounds.

Reply:
I'll add a CLA also. Thanks.